bitbots_docs/docs/manual/tutorials/Docs-How-To.rst

```diff
@@ -1,18 +1,14 @@
-===========
-How to Doku
-===========
+===============
+How to document
+===============
 
-Unter `doku.bit-bots.de <http://doku.bit-bots.de>`_ wird eine automatisch aktualisierte Doku zur Verfügung gestellt.
+Our documentation is published under  `doku.bit-bots.de <http://doku.bit-bots.de>`_ and will automatically be regenerated from the package `bitbots_misc/bitbots_docs <https://github.com/bit-bots/bitbots_misc>`_ and deployed by our `Jenkins CI <http://ci.bit-bots.de>`_.
```

Comment:
Why `bitbots_misc/bitbots_docs` and not `bitbots_meta/bitbots_tools/bitbots_docs`???

Reply:
Because that's where it probably was at one point. You pointed out correctly that the package now lives at `bitbots_meta/bitbots_tools/bitbots_docs`